### PR TITLE
feat: all server queries and chain header, new session subscription resolvers

### PR DESCRIPTION
**Pull request by yjkimjunior**
closing #132

### CODE REVIEW

**Comment by yjkimjunior** [comment]
actually missing the recent proposals/ref stuff.

**Comment by yjkimjunior** [comment]
ah nvm that would be for the polkassembly server. updated the datamodel anyway

**Comment by yjkimjunior** [comment]
CI: `Reinitialized existing Git repository in /builds/parity/nomidot/.git/
 fatal: Unable to create '/builds/parity/nomidot/.git/shallow.lock': No space left on device`

probably will retrigger in 10 minutes or so

File: back/server/src/resolvers/subscription.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+} from '../generated/prisma-client';
+import { Context, Selectors } from '../types';
+
+const chainBestBlockNumber = {
+  subscribe: (
+    parent: any,
+    { blockNumberSubscriptionWhereInput }: Selectors,
+    context: Context
+  ): (<T = BlockNumberSubscription>() => T) => {
+    return context.prisma.$subscribe
```
Comment (amaurymartiny) [inline back/server/src/resolvers/subscription.ts]: nice!

File: back/server/src/types.ts [hunk 2]
```diff
@@ -3,7 +3,11 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import { Prisma } from './generated/prisma-client';
+// import { PubSub } from 'graphql-yoga';
 
 export interface Context {
   prisma: Prisma;
+  // pubsub: PubSub;
```
Comment (amaurymartiny) [inline back/server/src/types.ts]: We can just remove commented code

**Pull request merged**